### PR TITLE
Update skeleton files

### PR DESCRIPTION
**Pull request by ziadhany**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,17 +1,29 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/skeleton for support or download.
```
Comment (keshav-space) [inline Makefile]:
```suggestion
# See https://github.com/nexB/cwe2 for support or download.
```

**Review by keshav-space — CHANGES_REQUESTED** [review]
Thanks @ziadhany ... just a minor nitpick for your consideration.

File: Makefile (hunk 1 above)
Comment (ziadhany) [reply Makefile]: Done

**Review by keshav-space — APPROVED** [review]
LGTM, thanks ++

**Pull request merged**